packages/core/core/src/core-api/routes/validation/utils.ts

```diff
@@ -95,7 +115,16 @@ export const safeSchemaCreation = (id: Internal.UID.Schema, callback: () => z.Zo
     // Replace the placeholder with the real schema
     safeGlobalRegistrySet(id, schema);
 
-    strapi.log.debug(`Schema ${transformedId} successfully created and registered`);
+    // Show completion for user content only
+    if (!isBuiltInSchema) {
+      const fieldCount = Object.keys((schema as any)?._def?.shape || {}).length || 0;
+      const schemaName = transformedId
+        .replace('Document', '')
+        .replace('Entry', '')
+        .replace(/([A-Z])/g, ' $1')
+        .trim();
+      strapi.log.info(`   ✅ "${schemaName}" schema created with ${fieldCount} fields`);
```

Comment:
Makes sense, but maybe we don't need the extra info in the log? Could keep it as ?

strapi.log.debug(`Schema ${schemaName} successfully created and registered`);